src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.{WorkflowCoercedInputs, NamespaceWithWorkflow}
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
+  sealed trait BackendValidationActorMessage
+  case class Validate(namespace: NamespaceWithWorkflow, wfInputs: Option[WorkflowCoercedInputs] = None, wfOptionsJson: Option[String] = None) extends BackendValidationActorMessage
+  case class ValidationResult(isSuccess: Boolean) extends BackendValidationActorMessage
```

Comment:
How about an ADT

```scala
sealed trait ValidationResult extends BackendValidationActorMessage
case object SuccessfulValidationResult extends ValidationResult
final case class FailureValidationResult(errors: List[String]) extends ValidationResult
```

which shows that this isn't handling failure correctly.  If the wf validation fails there's no logging nor a response to the `Validate` message.